javascript/chronometer.js

```diff
@@ -1,34 +1,53 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
-  }
+  
+    this intervalID = setInterval(() => {
+      this.currentTime = intervalCounter += 1;
+      
+      if(callback){
+      return callback;
+      }
+
+    }, 1000);
+   
+   
+    }
+}
 
   getMinutes() {
-    // ... your code goes here
+  
+    let Minutes = Math.floor this.currentTime /60;
+    return getMinutes;
```

Comment:
what happens if you do this is that you return the function you are executing. You should return Minutes to get the expected return value